### PR TITLE
Requisition list improvements

### PR DESCRIPTION
**Pull request by leslietilling**
## Purpose of this pull request

This pull request (PR) is the _add-on_ for #862 where new information about requisition lists was added to the config reference. This moves the information to incorporate it into the functional and task-based area for B2B configuration and additional linking for supported storefront functionality.

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/configuration/customers/requisition-lists.html
https://docs.magento.com/user-guide/customers/account-dashboard-requisition-lists.html
https://docs.magento.com/user-guide/stores/b2b-configure-requisition-lists.html (new)
https://docs.magento.com/user-guide/stores/b2b-features.html

whatsnew
Added the [Configuring Requisition Lists](https://docs.magento.com/user-guide/stores/b2b-configure-requisition-lists.html) topic for B2B configuration, as well as additional linking for supported storefront functionality.

### CODE REVIEW

File: src/stores/b2b-configure-requisition-lists.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+## Configure the maximum number of requisition lists
+
+1. On the _Admin_ sidebar, go to **Stores** > _Settings_ > **Configuration**.
+
+1. In the left panel, expand **Customers** and choose **Requisition Lists**.
+
+   ![]({% link images/images-b2b/config-customers-requisition-lists-general.png %}){: .zoom}
+   [_General setting_]({% link configuration/customers/requisition-lists.md %})
+
+1. For Number of Requisition Lists, enter the maximum number of requisition lists per customer account.
```
Comment (erikmarr) [inline src/stores/b2b-configure-requisition-lists.md:25]: Should Number of Requisition Lists be in italics or bold?

**Review by erikmarr — APPROVED** [review]
Looks good...just a minor comment.

File: src/stores/b2b-configure-requisition-lists.md (hunk 1 above)
Comment (leslietilling) [reply src/stores/b2b-configure-requisition-lists.md:25]: Should be bold... good catch!

**Comment by leslietilling** [comment]
running tests

**Pull request merged**